chimney-cats/src/main/scala/io/scalaland/chimney/cats/validated/TransformationError.scala

```diff
@@ -0,0 +1,21 @@
+package io.scalaland.chimney.cats.validated
+
+abstract class TransformationError(val info: String) {
+  def addPrefix(prefix: String): TransformationError
+}
+
+object TransformationError {
+  private case class Deep(message: String, path: NEC[String])
+      extends TransformationError(s"$message on ${path.toChain.iterator.mkString(".")}") {
```

Comment:
Could you provide access to the `path` from `TransformationError`?
It would be nice to have a possibility to interact directly with it.

Moreover, it seems like you reconstruct this message every time when you adding a prefix.
Correct me if I'm wrong.

So how about to define an `info` as `def` instead of `val` and to provide access to the `path`?